ROADMAP.md

```diff
@@ -8,28 +8,26 @@ We hope that the items listed below will inspire further engagement from the com
 Any dates listed below and the specific issues that will ship in a given milestone are subject to change but should give a general idea of what we are planning.
 See the [GitHub project boards](https://github.com/rook/rook/projects) for the most up-to-date issues and their status.
 
-## Rook Ceph 1.10
+## Rook Ceph 1.11
 
-The following high level features are targeted for Rook v1.10 (early August 2022). For more detailed project tracking see the [v1.10 board](https://github.com/rook/rook/projects/26).
+The following high level features are targeted for Rook v1.11 (January 2023). For more detailed project tracking see the [v1.11 board](https://github.com/rook/rook/projects/27).
 
-* Remove support for Ceph Octopus (support remains for Pacific and Quincy) [#10338](https://github.com/rook/rook/issues/10338)
-* Add command to the [krew plugin](https://github.com/rook/kubectl-rook-ceph) to analyze cluster health and advise on resolving common health issues [Krew #32](https://github.com/rook/kubectl-rook-ceph/issues/32)
-* Check for existing subvolumes before allowing a filesystem to be uninstalled [#9915](https://github.com/rook/rook/pull/9915)
 * Use more specific cephx accounts to better differentiate the source of Ceph configuration changes [#10169](https://github.com/rook/rook/issues/10169)
+* Enable restoring a cluster from the OSDs after all mons are lost [#7049](https://github.com/rook/rook/issues/7049)
 * Automate node fencing for application failover in some scenarios [#1507](https://github.com/rook/rook/issues/1507)
+* Support RBD mirroring across clusters with overlapping networks [#11070](https://github.com/rook/rook/issues/11070)
+* OSD encryption on partitions [#10984](https://github.com/rook/rook/issues/10984)
 * Object Store
   * Service account authentication with the Vault agent [#9872](https://github.com/rook/rook/pull/9872)
-  * Support for AWS Server Side Encryption (SSE) [#10318](https://github.com/rook/rook/pull/10318)
-  * Improvements to RGW Multisite configuration [#10323](https://github.com/rook/rook/pull/10323)
-
+  * Add alpha support for COSI (Container object storage interface) with K8s 1.25 [#7843](https://github.com/rook/rook/issues/7843)
+  * Support the immutable object cache [#11162](https://github.com/rook/rook/issues/11162)
```

Comment:
We will have cephfs encryption support https://github.com/ceph/ceph-csi/pull/3460 in cephcsi 3.8.0 , still have some dependencies with ceph